index.html

```diff
@@ -0,0 +1,136 @@
+            perferendis facere? Libero, aut aperiam?
+          </p>
+        </section>
+        <section id="what_you_should_already_know" class="main-section">
+          <h2>what you should already know</h2>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
+        </section>
+        <section id="JavaScript_and_Java" class="main-section">
+          <h2>JavaScript and Java</h2>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+        </section>
+        <section id="Hello_world" class="main-section">
+          <h2>Hello world</h2>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
+          <ul>
+            <li>Selam</li>
+            <li>Hello</li>
+            <li>Guten Morgon</li>
+            <li>Merhaba</li>
+            <li>Kaifa</li>
+          </ul>
+        </section>
+        <section id="Variables" class="main-section">
+          <h2>Variables</h2>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+            <code></code>
+            <code></code>
+            <code></code>
+            <code></code>
+            <code></code>
+          </p>
+        </section>
+      </main>
+    </div>
+
+    <!-- <p>Hello World, we are software developers!</p>
+    <p></p> -->
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
Please remove this comment. It makes your code cleaner and it appears that it is copied from other sources.